service/evidence/evidence_store.go

```diff
@@ -202,9 +202,34 @@ func (svc *Service) ListEvidences(ctx context.Context, req *evidence.ListEvidenc
 
 	res = new(evidence.ListEvidencesResponse)
 
+	var query []string
+
+	// Apply filter options
+	if req.GetFilter() != nil {
+		filter := req.GetFilter()
```

Comment:
There is a nice litte Go idiom to make this a one-liner: `if filter := req.GetFilter(); filter != nil {`

Reply:
Very handy!